openldap-mixin/main.libsonnet

```diff
@@ -0,0 +1,83 @@
+local alerts = import './alerts.libsonnet';
+local dashboards = import './dashboards.libsonnet';
+local datasources = import './datasources.libsonnet';
+local g = import './g.libsonnet';
+local panels = import './panels.libsonnet';
+local targets = import './targets.libsonnet';
+local variables = import './variables.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+local link = g.dashboard.link;
+
+{
+  new(
+    filteringSelector,
+    groupLabels=['job'],
+    instanceLabels=['instance'],
+    dashboardNamePrefix='',
+    dashboardTags=[uid],
+    uid,
+  ): {
+    local this = self,
+    config: {
+      groupLabels: groupLabels,
+      instanceLabels: instanceLabels,
+      filteringSelector: filteringSelector,
+      dashboardTags: dashboardTags,
+      uid: uid,
+      dashboardNamePrefix: dashboardNamePrefix,
+      dashboardPeriod: 'now-1h',
+      dashboardTimezone: 'default',
+      dashboardRefresh: '1m',
+
+      // Alert thresholds
+      alertsWarningConnectionSpike: 100,
+      alertsWarningHighSearchOperationRateSpike: 100,
+      alertsCriticalDialFailureRateIncrease: 50,
+      alertsCriticalBindFailureRateIncrease: 50,
+
+      // logs lib related
+      enableLokiLogs: false,
```

Comment:
it looks like you're following the pattern that was used for the active directory library, which i think was special because it was a library. i would follow something closer to the helloworld example or what i did with istio (this file could be a lost smaller, and the extra config stuff could go right in the config file, links could go right in a links file)

Reply:
I removed part of the excess bits. Still getting errors as I was removing everything though trying to follow that simplified approach. I'll give it another shot to see if I can reduce it further.

Reply:
yeah. you shouldn't need to pass in a whole new config here. it's probably "fine" that you're creating the `links` on the fly, but might be a little nicer to put them in their own file.

Reply:
Good to resolve